queries/markdown_inline/highlights.scm

```diff
@@ -0,0 +1,74 @@
+;; From MDeiml/tree-sitter-markdown
+[
+  (code_span)
+  (link_title)
+] @text.literal
+
+[
+  (emphasis_delimiter)
+  (code_span_delimiter)
+] @punctuation.delimiter
+
+(emphasis) @text.emphasis
+
+(strong_emphasis) @text.strong
+
+[
+  (link_destination)
+  (uri_autolink)
+] @text.uri
+
+[
+  (link_label)
+  (link_text)
+  (image_description)
+] @text.reference
+
+[
+  (backslash_escape)
+  (hard_line_break)
+] @string.escape
+
+(image "!" @punctuation.delimiter)
+(image "[" @punctuation.delimiter)
+(image "]" @punctuation.delimiter)
+(image "(" @punctuation.delimiter)
+; (image ")" @punctuation.delimiter)
+
+(inline_link "[" @punctuation.delimiter)
+(inline_link "]" @punctuation.delimiter)
+(inline_link "(" @punctuation.delimiter)
+; (inline_link ")" @punctuation.delimiter)
```

Comment:
why is this commented ?
also i think you can do:
```
(inline_link [ "[" "]" "(" ")" ] @punctuation.delimiter)
```

Reply:
I just took this from the old markdown queries without changing it. I think it was commented because there was a bug where tree-sitter crashed and it was determined that commenting this out would solve that, but I'm not sure it's still relevant.

I'm gonna change to your more concise version though.